ee/clickhouse/sql/events.py

```diff
@@ -167,7 +167,8 @@
 """
 
 NULL_SQL = """
-SELECT toUInt16(0) AS total, {interval}(toDateTime('{date_to}') - number * {seconds_in_interval}) as day_start from numbers({num_intervals})
+SELECT toUInt16(0) AS total, {interval_method}(toDateTime('{date_to}') - INTERVAL number {interval}) AS day_start
```

Comment:
Note: Using `toInterval` methods here is preferred over `INTERVAL` since it allows passing parameters in as... parameters.

https://clickhouse.tech/docs/en/sql-reference/functions/type-conversion-functions/#function-tointerval

Reply:
Good catch, I didn't know about this family of functions, but they do look cleaner!